### PR TITLE
Bunch of test related things

### PR DESCRIPTION
**Pull request by silverlight6**
This is my work from the last 2 days. There is a lot so have fun.

### CODE REVIEW

File: Models/MCTS_Util.py [hunk 1]
```diff
@@ -6,27 +6,52 @@
 
 def create_default_mapping():
     mappings = []
+    mask = []
 
-    for a in range(38):
-        for b in range(37):
+    for b in range(37):
```
Comment (Lobotuerk) [inline Models/MCTS_Util.py:11]: For the future, avoiding these kind of magic numbers is a must. Is this board size?

File: Models/MCTS_Util.py [hunk 2]
```diff
@@ -6,27 +6,52 @@
...
+                mask.append(1)
             else:
                 mappings.append(f"5_{a}_{b}")
+                if a == b:
+                    mask.append(0)
+                else:
+                    mask.append(1)
 
-        for b in range(10):
+    for b in range(10):
```
Comment (Lobotuerk) [inline Models/MCTS_Util.py:23]: And bench size

**Review by Lobotuerk — COMMENTED** [review]
Looking better. You should also test observations by "de-inputifiying" an observation after you translated it into input

File: Models/MCTS_Util.py (hunk 1 above)
Comment (silverlight6) [reply Models/MCTS_Util.py:11]: It's the board size plus the bench size. The 38 adds one for the sell action. It's a repeat of what is in the action class basically.

Comment (Lobotuerk) [reply Models/MCTS_Util.py:11]: So you see how having 2 constants like (board size and bench size) could make things way more readeable

File: Models/MCTS_Util.py (hunk 2 above)
Comment (silverlight6) [reply Models/MCTS_Util.py:23]: I'll move from Simulator config to main config.

**Pull request merged**